pages/referencing-the-manuscript.xml

```diff
@@ -17,62 +17,187 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+            more references listed which are not a range. Please do not use other numbering systems
+            or spaces within these attributes.</p>
+
+
+         <p>Beside the basic referencing structures (<gi>div</gi>) you may refer to parts of the
+            text of the manuscript using references to <gi>pb</gi>, <gi>cb</gi> and <gi>lb</gi>.
+            Although for these to be resolvable the text needs to be present and encoded, their
+            semantic value with reference to the selected passage remains even when the actual
+            elements and text is not present and the reference is intended to the material object.
+            The following are possible references: <list type="ordered">
+               <item>a folio, e.g. <code>1</code> (<tag>div[@type='folio'][@n]</tag>)</item>
+               <item>a page, e.g. <code>1r </code>(<tag>pb[@n]</tag>)</item>
+               <item>a column, e.g. <code>1va</code> (<tag>pb[@n]</tag>,<tag>cb[@n]</tag>)</item>
+               <item>a line, e.g. <code>2.1</code>
+                     (<tag>div[@type='folio'][@n]</tag>,<tag>lb[@n]</tag>). Note that this is the
+                  same very generic referencing system of any numbered division declared in the
+                  document, if can refer to a line in a folio, but also to a chapter in a book or a
+                  day in a month, depending on what is declared in the file.</item>
+               <item>a line, e.g. <code>2</code> (<tag>lb[@n]</tag>), if the folio structure is not
+                  present.</item>
+               <item>a line, eventually in a specific page and column, e.g. <code>2ra1</code>
+                     (<tag>pb[@n]</tag>,<tag>cb[@n]</tag>, <tag>lb[@n]</tag>)</item>
+            </list> This assumes consistent numbering.</p>
+         <p> It is possible to use the same syntax for a reference to a page in one of more defined
+            paging systems, e.g. based on different editions. e.g. <code>1[frisk]</code>
+               (<tag>pb[@n][@corresp]</tag>). </p>
+
+         <egXML xmlns="http://www.tei-c.org/ns/Examples">
+            <locus from="1v1" to="2ra6">the text begins on 1v in line 1 and ends on 2ra in line 6
+            </locus>
+         </egXML>
+         <p>A list of individual references at page level (without line indication) can be given in
+               <att>target</att>
          </p>
          <egXML xmlns="http://www.tei-c.org/ns/Examples">
-<locus target="#28v #33r #49r #50v #55v #82r #82v #83r #83v #84v #85r"/>
-</egXML>
-         <p>Please DO NOT use unknown as a value for any of these attributes.
-    Simply do not enter the attribute if its value is not known.</p>
+            <locus target="#28v #33r #49r #50v #55v #82r #82v #83r #83v #84v #85r"/>
+         </egXML>
+         <p>Please DO NOT use unknown as a value for any of these attributes. Simply do not enter
+            the attribute if its value is not known.</p>
```

Comment:
Could this be deleted? I think no one has done that for a long time...

Reply:
I have done some investigation of usage of the attributes in locus to find out the most wild wildness. I do not think this will hurt anyone who knows already, but perhaps will stop someone from having that idea in the next few years.